mindjunkies/forums/views.py

```
@@ -177,3 +178,65 @@ def post(self, request, *args, **kwargs):
+    model = Reply
+    template_name = 'forums/partials/like_reply.html'
+    context_object_name = 'reply'
+      
+
+
+
+
+
+
+
+   
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)